components/Game.tsx

```diff
@@ -55,6 +73,80 @@ export default class Game extends React.Component<{}, GameState> {
         }
     }
 
+    checkMatch = (data: Array<Square>) => {
+        console.log("check match");
+        let temp: string;
+        let matches = 1; 
+        let i: number;
+        let hasMatch = false;
+
+        temp = data[0].type;
+        hasMatch = false;
+
+        // horizontal check
+        for (i = 1; i < 64; i++) {
```

Comment:
Store any numbers that represent the number of rows or columns or fixed numbers as constants at the top of the game class so they can be easily reused.